bokeh/embed.py

```diff
@@ -34,15 +42,19 @@ def components(plot_object, resources):
     Args:
         plot_object (PlotObject) : Bokeh object to render
             typically a Plot or PlotContext
-        resources (Resources, optional) : BokehJS resources config
-
+        resources : Deprecated argument
     Returns:
         (script, div) : UTF-8 encoded
 
     '''
+    
+    if resources is not None:
+        warn('the ``resources`` argument is deprecated and will be removed in'
+             'a future version.', DeprecationWarning, stacklevel=2)
+    
```

Comment:
Offered a comment here https://github.com/bokeh/bokeh/pull/2236#issuecomment-97509813.  Quickly, it's source mapping not just the CoffeeScript from Bokeh, but the entire browserified file.  There are a few options, some more palatable than others.